config/locales/de.yml

```diff
@@ -18,9 +20,12 @@ de:
     my_contacts: "Mein Adressbuch"
     select_from_contacts: "Aus Kontakten auswählen"
     save: "Speichern"
+    add_interaction: "Interaktion hinzufügen"
```

Comment:
German question: Maybe `Treffen` or `Begegnung` is less formal than INTERAKTION? (yes, these words need to be screamed lol)

Reply:
yes but again, i like the extreme german but we can change it to normal stuff yes:(

Reply:
Let's keep it this way for now and then see down the line 🌈